### PR TITLE
Release notes for 8.0.0-beta1

### PR DESCRIPTION
**Pull request by logstashmachine**
**PREVIEW:** https://logstash_13373.docs-preview.app.elstc.co/guide/en/logstash/8.0/logstash-8-0-0-beta1.html

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -3,11 +3,54 @@
 
 coming[8.0.0]
 
+* <<logstash-8-0-0,Logstash 8.0.0>>
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:6]:
```suggestion
```

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -5,9 +5,51 @@ coming[8.0.0]
...
+      "@version" => "1"
+}
+--------------------------------------------------------------------------------
+
+NOTE: These prs below are also in 7.15/7.16, we can delete these entries before v8.0.0-beta1 release
+* Secure API https://github.com/elastic/logstash/pull/13308[#13308]
+* geoip integrate air-gapped bootstrap script https://github.com/elastic/logstash/pull/13104[#13104]
+* Add deprecation warnings for JAVA_HOME/… https://github.com/elastic/logstash/pull/13365[#13365]
+* Fixes to build and run Logstah on JDK 17 https://github.com/elastic/logstash/pull/13306[#13306]
+
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc:51]: I think we're missing some content here around dropping support for java 8. I can work with @karenzone on the language here

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -5,9 +5,58 @@ coming[8.0.0]
...
+       "message" => "test",
+          "type" => "stdin",
+      "@version" => "1"
+}
+--------------------------------------------------------------------------------
+
+NOTE: These prs below are also in 7.15/7.16, we can delete these entries before v8.0.0-beta1 release
+* Secure API https://github.com/elastic/logstash/pull/13308[#13308]
+* geoip integrate air-gapped bootstrap script https://github.com/elastic/logstash/pull/13104[#13104]
+
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:58]:
```suggestion
```

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -5,9 +5,54 @@ coming[8.0.0]
...
+* <<logstash-8-0-0-beta1,Logstash 8.0.0-beta1>>
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-0-0-beta1]]
+=== Logstash 8.0.0-beta1 Release Notes
+
+==== Breaking Changes
+
+[[java-11-minimum]]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:17]:
```suggestion
[[rn-java-11-minimum]]
```

Comment (karenzone) [reply docs/static/releasenotes.asciidoc:17]: We can't reuse anchors, and I don't want to waste them on release notes.

File: docs/static/releasenotes.asciidoc [hunk 5]
```diff
@@ -5,9 +5,54 @@ coming[8.0.0]
...
+Starting from Logstash 8.0, the minimum required version of Java to run Logstash is Java 11. By default, Logstash will
+run with the bundled JDK, which has been verified to work with each specific version of Logstash, and generally
+provides the best performance and reliability.
+
+==== New features and enhancements
+
+[[nanosecond-precision]]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:24]:
```suggestion
[[rn-nanosecond-precision]]
```

**Review by karenzone — COMMENTED** [review]
Comments inline. Making another pass.

File: docs/static/releasenotes.asciidoc (hunk 4 above)
Comment (jsvd) [reply docs/static/releasenotes.asciidoc]: @karenzone this file has a few other anchors that aren't prefixed by `rn-`, should we change them as well?

File: docs/static/releasenotes.asciidoc [hunk 6]
```diff
@@ -5,9 +5,58 @@ coming[8.0.0]
...
+    "@timestamp" => 2021-10-31T22:32:34.747968Z,
+          "host" => "logstash.lan",
+         "nanos" => "Nanos: 747968000",
+       "message" => "test",
+          "type" => "stdin",
+      "@version" => "1"
+}
+--------------------------------------------------------------------------------
+
+
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:59]: Should we add a note about which 7.x release the beta-1 plugins align with?  (See lines 103-104 in this file for reference.)

**Review by karenzone — COMMENTED** [review]
Pushed a change and left one more comment wrt plugins. We're getting close!

**Review by karenzone — APPROVED** [review]
LGTM!

**Pull request merged**